doc/source/whatsnew/v1.6.0.rst

```diff
@@ -100,6 +100,7 @@ Deprecations
 
 Performance improvements
 ~~~~~~~~~~~~~~~~~~~~~~~~
+- Performance improvement in :meth:`.GroupBy.cumprod` for extension array dtypes (:issue:`37493`)
```

Comment:
This also now uses int64 instead of float64 for the numpy dtypes? So that also changes behaviour in those cases regarding overflow?

Reply:
Yes, should we mention this in the whatsnew?

Reply:
I think so, yes. Maybe as notable bug fix, as it has some behaviour change?

Reply:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/61934744/188241996-b2bb4ffc-5ada-443b-9f15-856f09a89e79.png">